product_stock_state/models/__init__.py

```diff
@@ -1,3 +1,8 @@
 # -*- coding: utf-8 -*-
 
-from . import product
+from . import res_company
+from . import sale_config_settings
```

Comment:
FYI, in v11 all settings are in the same model.
Not blocking, but for later PR we could call this file config_settings.py, then migration'll be smarter